docs/src/robin_dict.md

```diff
@@ -0,0 +1,36 @@
+# RobinDict
+
+`RobinDict` is simply a dictionary which uses Robin Hood hashing technique in the background for performing standard `Dict` operations.
+
+The function-calls of `RobinDict` replicates that of `Dict`.
+
+Examples:
+
+```julia
```

Comment:
Does this repo support and run doctest? If not, should probably at least use `julia-repl` here.

Reply:
this repo supports doctests, i had previously committed to the repo [documentation](https://github.com/JuliaCollections/DataStructures.jl/pull/492)  and used `julia` there, so I guess it should work fine.